service/kg-search/src/main/java/eu/ebrains/kg/search/KgSearchApplication.java

```diff
@@ -56,5 +57,10 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http, JwtUserInfoCon
         return http.build();
     }
 
+    @Bean
+    public RestTemplate restTemplate() {
```

Comment:
I just stumbled upon this: https://www.baeldung.com/rest-template#rest-template-is-deprecated - maybe we should try to use webclient instead